flocx_market/db/sqlalchemy/api.py

```diff
@@ -41,7 +41,7 @@ def drop_db():
 def offer_get(marketplace_offer_id, context):
 
     return get_session().query(models.Offer).filter_by(
-            marketplace_offer_id=marketplace_offer_id).one_or_none()
+            marketplace_offer_id=marketplace_offer_id).first()
```

Comment:
Ah, we're using `one_or_none()` now, after a change Lars made.

Reply:
sounds good